Swift Chat Demo 2/ConversationListViewController.swift

```diff
@@ -48,18 +48,16 @@ class ConversationListViewController: UIViewController {
     }
     
     override func viewWillAppear(_ animated: Bool) {
+        
         self.fetchConversations(successBlock: {
             print("Fetched Conversations")
         }) { (error) in
             SVProgressHUD.showError(withStatus: error.localizedDescription)
         }
-    }
-    
-    override func viewDidAppear(_ animated: Bool) {
         subscribeUserChannel()
     }
     
-    override func viewDidDisappear(_ animated: Bool) {
+    override func viewWillDisappear(_ animated: Bool) {
```

Comment:
Would you mind help adding `super` to functions like `viewWillDisappear`, `viewWillAppear`... etc? From the doc, we must call super for those functions.

https://developer.apple.com/documentation/uikit/uiviewcontroller/1621485-viewwilldisappear